### PR TITLE
Fixes #17146: Update release info about 6.1

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/17146

### CODE REVIEW

File: release-data/changelogs/6.1/index.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+== What's new?
+
+=== New security features (through plugins)
+
+* CIS policy pack: Allows to apply pre-made CIS policies in one click using Rudder
+* CVE management: Reports CVEs affecting installed packages
+* OpenSCAP: Runs OpenSCAP audits, collects and exposes them in the web interface
+
+=== Polishing the web interface
```
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/index.adoc:11]: I think you can remove this :) apart "New group page UI"

Comment (VinceMacBuche) [reply release-data/changelogs/6.1/index.adoc:11]: and new group page needs to be added!!

File: release-data/changelogs/6.1/main.adoc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+= Change logs for Rudder 6.0
```
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/main.adoc:1]: 6.1

**Comment by amousset** [comment]
PR updated with a new commit

**Pull request merged**